### PR TITLE
MAM-2600-users-foryou-feed-incorporates-their-subscribed-channels

### PR DESCRIPTION
**Pull request by jtomchak**
* User's subscribed channels will be added to their ForYou feed
   * Note. There is an edge case where you subscribe to a channel, you won't immediately see it in your ForYou feed until it's updated server side. That happens every 5 minutes.

### Example of seeing #News Channel in my ForYou feed
RocketSim_Screenshot_iPhone_14_Pro_2023-09-15_10 11 46
RocketSim_Screenshot_iPhone_14_Pro_2023-09-15_10 11 31

### CODE REVIEW

File: app/lib/mammoth/channels.rb [hunk 1]
```diff
@@ -3,9 +3,41 @@ module Mammoth
   class Channels
     class NotFound < StandardError; end
 
+    GO_BACK = 24 # number of hours back to fetch statuses
     ACCOUNT_RELAY_AUTH = "Bearer #{ENV.fetch('ACCOUNT_RELAY_KEY')}"
```
Comment (jtomchak) [inline app/lib/mammoth/channels.rb:7]: originally I thought, 2 hours should be good to run the cron with overlap. But I forgot about past posts where engagement goes up over time. If the post was older than a couple hours and got a lot of engagement, it wouldn't have gotten picked up. So we want to reach back 24 hours, _and then_ filter for engagement

File: config/sidekiq.yml [hunk 2]
```diff
@@ -71,11 +71,11 @@
...
   channel_statuses_scheduler:
     queue: scheduler
-    interval: '5m'
+    interval: '2m'
     class: Scheduler::ChannelMammothStatusesScheduler
   for_you_statuses_scheduler:
     queue: scheduler
-    interval: '2m'
+    interval: '5m'
     class: Scheduler::ForYouStatusesScheduler
```
Comment (jtomchak) [inline config/sidekiq.yml:79]: accidentally reversed those numbers earlier

**Review by rileyhCode — APPROVED** [review]
LGTM.
Can you pls file a ticket for this? Would it be possible to do something like "trigger a ForYou rebuild when the user changes their channel un/subscribe settings"?

> There is an edge case where you subscribe to a channel, you won't immediately see it in your ForYou feed until it's updated server side.

**Comment by jtomchak** [comment]
> LGTM. Can you pls file a ticket for this? Would it be possible to do something like "trigger a ForYou rebuild when the user changes their channel un/subscribe settings"?
> 
> > There is an edge case where you subscribe to a channel, you won't immediately see it in your ForYou feed until it's updated server side.

added https://linear.app/theblvd/issue/MAM-2668/subscribing-to-a-channel-doesnt-immediately-add-it-to-foryou

**Pull request merged**